core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -97,7 +97,7 @@
  * @param <ReqT> type of message sent one or more times to the server.
  * @param <RespT> type of message received one or more times from the server.
  */
-@DoNotMock("Use InProcessTransport and make a fake server instead")
+@DoNotMock("Use InProcessServerBuilder and make a full featured in process server instead")
```

Comment:
Precision is the wrong thing here.  Naming the exact class they use is a better usability story.  People who run into this annotation probably don't know what they should do.  

Even better would be pointing to an example class, but the change as is is better than previously.